manifests/0000_90_cluster-svcat-apiserver-operator_01_remover_job.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: openshift-service-catalog-apiserver-remover
+  namespace: openshift-service-catalog-removed
```

Comment:
This is not going to work, because the namespace will still be there after the job runs, right?  Can you live in someone else's namespace instead?  Some choices [here][1].  Maybe just the `openshift` namespace?

[1]: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-svcat-apiserver-operator/74/pull-ci-openshift-cluster-svcat-apiserver-operator-master-e2e-aws/218/artifacts/e2e-aws/must-gather/registry-svc-ci-openshift-org-ci-op-di0k4j40-stable-sha256-dd03a9e0ac7b6e710037d8f0d0a5001b47c27bf859e49b0256e8ac54f5bd8198/namespaces/

Reply:
Hmm, we may just want to create a new `openshift-remover` namespace with a cluster-admin service account set up for these sorts of things?  I'll float something like that over on the CVO side...

Reply:
@wking we asked and one option was to use another namespace but that seems really odd to have a service-catalog remover job running in some other operator's namespace. We also asked if there was already an existing namespace for jobs, but there wasn't. We decided to create one at least for the time being until there is a better solution for jobs. @smarterclayton didn't want to have to explain why these things are still around or what they were for. 

> you would have a job
> your job should be specific to your component or operator
> it would probably be best if you chose a namespace coupled to the operator
> i.e. i’d rather have openshift-service-catalog lying around than openshift-SOME_WIERD_THING_I_HAVE_TO_EXPLAIN
> a future release could drop the namespace
> call it openshift-service-catalog-removed

That was the guidance we went with. There is more in that thread I can send you, but that's a summary.